src/remotestorage.js

```diff
@@ -721,7 +744,7 @@
       }
 
       function featureSupported(name, success) {
-        self.log("[RemoteStorage] [FEATURE "+name+"]" + success ? "":" not"+" supported");
+        self.log("[RemoteStorage] [FEATURE "+name+"]" + (success ? "" : " not") + " supported");
```

Comment:
This looks like an actual fix for a bug here, right? I think there's a space char too much in front of the colon, but other than that it should probably go in a pull request to master.